README.md

```diff
@@ -252,6 +252,32 @@ host{["fqdn"<br/>"simple"<br/>"address"]} | Outputs either the FQDN hostname, th
 </Configuration>    
 ```    
 
+**YAML**
+```yaml
+    rootLogger.level = INFO
+    rootLogger.appenderRef.gelf.ref = GelfAppender
+
+    appender.gelf.type = Gelf
```

Comment:
proper yaml would make use of prefixes to not duplicate `appender.gelf` and `:` Care to update your sample?

Reply:
Updated the two examples